tests/unit/test_vendor.py

```diff
@@ -71,44 +38,3 @@ def test_inject_environment_variables(self):
         assert parsed_dict["list"][1] == {}
         assert parsed_dict["bool"] is True
         assert parsed_dict["none"] is None
-
-
-@pytest.mark.parametrize('dt, content', [
-    (   # Date.
-        datetime.date(1992, 8, 19),
-        '1992-08-19',
-    ),
-    (   # Naive time.
-        datetime.time(15, 10),
-        '15:10:00',
-    ),
-    (   # Aware time in UTC.
-        datetime.time(15, 10, tzinfo=pytz.UTC),
-        '15:10:00Z',
-    ),
-    (   # Aware local time.
-        datetime.time(15, 10, tzinfo=pytz.FixedOffset(8 * 60)),
-        '15:10:00+08:00',
-    ),
-    (   # Naive datetime.
-        datetime.datetime(1992, 8, 19, 15, 10),
-        '1992-08-19T15:10:00',
-    ),
-    (   # Aware datetime in UTC.
-        datetime.datetime(1992, 8, 19, 15, 10, tzinfo=pytz.UTC),
-        '1992-08-19T15:10:00Z',
-    ),
-    (   # Aware local datetime.
-        datetime.datetime(1992, 8, 19, 15, 10, tzinfo=pytz.FixedOffset(8 * 60)),
-        '1992-08-19T15:10:00+08:00',
-    ),
-])
-def test_token_date(dt, content):
-    token = create_primitive_token(dt)
-    assert token == tokens.Token(tokens.TYPE_DATE, content)
-
-
-def test_dump_nonascii_string():
-    content = 'name = "Stažené"\n'
-    toml_content = contoml.dumps(contoml.loads(content))
-    assert toml_content == content
```

Comment:
I think these tests in `test_vendor.py` should be kept, but rewrote to use tomlkit.

Reply:
Sure, thanks for the review.